var/spack/repos/builtin/packages/fabtests/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Fabtests(AutotoolsPackage):
+    """Fabtests provides a set of examples that uses libfabric"""
+
+    homepage = "http://libfabric.org"
+    url      = "https://github.com/ofiwg/fabtests/archive/v1.5.3.tar.gz"
```

Comment:
If you download from https://github.com/ofiwg/fabtests/releases/download/v1.5.3/fabtests-1.5.3.tar.gz instead of https://github.com/ofiwg/fabtests/archive/v1.5.3.tar.gz you can drop all of the Autotools dependencies. GitHub usually creates a source tarball with each release, and developers have the option of uploading their own tarball that contains generated files like `configure`. This way, the package has fewer dependencies and can be installed more quickly.

Reply:
Changed, reinstalled - confirmed working without the extra autotools deps